themes/tiddlywiki/vanilla/base.tid

```diff
@@ -3234,4 +3237,16 @@ select {
 
 .tc-tiny-v-gap-bottom {
 	margin-bottom: 3px;
-}
\ No newline at end of file
+}
+
+.tc-big-v-gap {
+	margin-top: 1em;
```

Comment:
I'm not sure if it's an artefact of github.com, but it looks like there's some uneven indenting here, and also on line 3250.

Reply:
This was my fault, some lines were spaced using four spaces instead of a tab, it's fixed now.